docs/javascript/v2/advanced-features/peer-connection-quality.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Peer Connection Quality
+nav: 5.06
+---
+
+You can get a score of current user's connection quality when in preview, post join connection quality will be available for all peers.
+The score ranges from 1-5, 5 being the hightest(good network health) and 1 being the lowest(bad network health).
+-1 indicates the sdk is not able to determine the network health within reasonable time.
```

Comment:
should we use -1 or 0. currently we are returning -1 in the published sdk.

Reply:
-1, as that is what server sends for undefined. the lowest from server is 0 which means disconnected(no traffic on ice)